iree/hal/local/elf/CMakeLists.txt

```diff
@@ -27,19 +27,24 @@ iree_cc_library(
   PUBLIC
 )
 
-iree_cc_test(
+iree_cc_binary(
   NAME
-    elf_module_test
+    elf_module_run
   SRCS
-    "elf_module_test.cc"
+    "elf_module_run.c"
```

Comment:
let's still call this _test_binary or something - it's not a general tool for running elfs

Reply:
Changed the source file as `elf_module_test_main.c` and the binary as `elf_module_test_binary`